app/connector/BindingTariffClassificationConnector.scala

```diff
@@ -29,21 +28,25 @@ import models.CaseStatus._
 import models.EventType.EventType
 import models._
 import models.request.NewEventRequest
-import scala.concurrent.{ ExecutionContext, Future }
+import scala.concurrent.{ExecutionContext, Future}
 import utils.JsonFormatters._
+import uk.gov.hmrc.http.HttpReads.Implicits._
```

Comment:
This fixes deprecation warnings about `HttpReads`. It also gives us instances for things like `HttpReads[Unit]` so that we can ignore responses if we only care about the status as described on [this page](https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?spaceKey=TEC&postingDay=2020%2F6%2F30&title=Http-verbs+update)